q2_feature_table/_summarize/_visualizer.py

```diff
@@ -46,12 +46,22 @@ def tabulate_seqs(output_dir: str, data: DNAIterator) -> None:
     shutil.copy(js, os.path.join(output_dir, 'js', 'tsorter.min.js'))
 
 
-def summarize(output_dir: str, table: biom.Table) -> None:
-    number_of_samples = table.shape[1]
-    number_of_features = table.shape[0]
+def summarize(output_dir: str, table: pd.DataFrame,
```

Comment:
Got it swapped and reused some things that were already calculated to build the jsonp. I think was because I originally built this as a new visualizer, and ported the code over. Thought it'd take more to redo what I had in `biom` than to convert everything else to `pd`, but after just swapping it back it wasn't, as everything was already being computed further down the line. On a quick test `biom` is faster on the Moving Pictures table. 
![image](https://cloud.githubusercontent.com/assets/2638727/22522619/7e2edef2-e879-11e6-9846-1f8ec652ff19.png)